README.rst

```diff
@@ -43,6 +44,12 @@ Usage
     schedule.every().wednesday.at("13:15").do(job)
     schedule.every().minute.at(":17").do(job)
 
+    """ Scheduling with argument in function """
+    def function_job(name):
```

Comment:
```suggestion
    def job_with_argument(name):
```
Include the description into the function name, so that the comment can be removed?